x-pack/plugins/observability/public/components/app/section/uptime/index.tsx

```diff
@@ -182,7 +182,7 @@ function UptimeBarSeries({
       />
       <Axis
         id="y-axis"
-        showGridLines
+        gridLine={{ visible: false }}
```

Comment:
I haven't looked at the updated API for the charts, but just looking at this code makes me think we were previously showing grid lines, and the new change would hide them. Am I off track here?

```suggestion
        gridLine={{ visible: true }}
```

A similar change was made in logs https://github.com/elastic/kibana/pull/75700/files#diff-c4bcb866f92a9fd9bbfa6a2e4402e3d5R153 and the prop value was set to `true`.

Reply:
@nickofthyme Does the suggested change look correct?

Reply:
Yeah that looks like a mistake I'll correct both cases

Reply:
committed suggestion